.gitignore

```diff
@@ -14,4 +14,6 @@ testing/coverage.txt
 pf/coverage.out
 pf/tests/coverage.out
 
-pulumi-hcl-lint
\ No newline at end of file
+pulumi-hcl-lint
+
+schema-tpsdkv2.json
```

Comment:
Why are we adding this. We should fix the test that erroneously generates this file instead of ignoring it.